src/popup/components/Modals/SpendSuccess.vue

```diff
@@ -78,12 +79,15 @@ export default {
     ...mapState('fungibleTokens', ['availableTokens']),
     ...mapGetters(['getTxAmountTotal', 'getTxSymbol', 'getExplorerPath']),
     ...mapGetters('names', ['getPreferred']),
+    isAE() {
```

Comment:
In order to remain consistent, I'd suggest not to use two capital letters next to each other in function names. So it should be `isAe` instead of `isAE`.